### PR TITLE
2023-05-26 Rainer Stropek

### PR DESCRIPTION
**Pull request by allenwyma**

### CODE REVIEW

File: _episodes/interviews/2023-05-12-john-nunley.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ length: "62503070"
 #reddit: (leave blank on initial publish, amend with link and uncomment this line after Reddit thread has been posted)
 ---
 
-Allen Wyma and [Zeeshan Ali Khan](https://www.linkedin.com/in/zeenix/) talk with [John Nunley](https://github.com/notgull), contributor to the [smol](https://github.com/smol-rs/smol) async runtime.
+Allen Wyma and [Zeeshan Ali Khan](https://toot.cat/@zeenix) talk with [John Nunley](https://github.com/notgull), contributor to the [smol](https://github.com/smol-rs/smol) async runtime.
```
Comment (jonhoo) [inline _episodes/interviews/2023-05-12-john-nunley.md:10]: I normally prefer for other changes to go into separate PRs, but okay for now.

File: _episodes/interviews/2023-05-26-rainer-stropek.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+---
+title: "CoderDojo and Rust Linz with Rainer Stropek"
+date: 2023-05-26T15:22:00Z
```
Comment (jonhoo) [inline _episodes/interviews/2023-05-26-rainer-stropek.md:3]:
```suggestion
date: 2023-05-27T16:14:00Z
```

File: _episodes/interviews/2023-05-12-john-nunley.md [hunk 3]
```diff
@@ -52,4 +52,4 @@ Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
 
 Show Notes: [Plangora](https://twitter.com/plangora)
 
-Hosts: [Allen Wyma](https://twitter.com/allenwyma)
+Hosts: [Allen Wyma](https://twitter.com/allenwyma) and Zeeshan Ali Khan
```
Comment (jonhoo) [inline _episodes/interviews/2023-05-12-john-nunley.md:55]:
```suggestion
Hosts: [Allen Wyma](https://twitter.com/allenwyma) and [Zeeshan Ali Khan](https://toot.cat/@zeenix)
```

File: _episodes/interviews/2023-05-26-rainer-stropek.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Intro Theme: [Aerocity](https://twitter.com/AerocityMusic)
+
+Audio Editing: [Plangora](https://twitter.com/plangora)
+
+Hosting Infrastructure: [Jon Gjengset](https://twitter.com/jonhoo/)
+
+Show Notes: [Plangora](https://twitter.com/plangora)
+
+Hosts: [Allen Wyma](https://twitter.com/allenwyma) and Zeeshan Ali Khan
```
Comment (jonhoo) [inline _episodes/interviews/2023-05-26-rainer-stropek.md:51]:
```suggestion
Hosts: [Allen Wyma](https://twitter.com/allenwyma) and [Zeeshan Ali Khan](https://toot.cat/@zeenix)
```

File: _episodes/interviews/2023-05-26-rainer-stropek.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+- Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+- Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+- Github: [@rustacean-station](https://github.com/rustacean-station/)
+- Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+## Timestamps
+
+- [@0:00] - Introduction to Rust Linz
+- [@3:23] - How did Rainer get into teaching kids in Austria how to code
+- [@5:17] - CoderDojo's vision in creating a coding club
```
Comment (jonhoo) [inline _episodes/interviews/2023-05-26-rainer-stropek.md:25]: I'm a little on the fence about this episode in terms of its connection to Rust. There's some Rust in here, which is good, but otherwise it's a more general discussion about coding clubs + about this specific one. In this instance I think it's okay because teaching kids to code is a topic that's near and dear to my heart, though just a heads up that we should try to keep things focused on Rust rather than programming as a more general topic.

cc @bstrie

**Review by jonhoo — APPROVED** [review]

**Comment by bstrie** [comment]
I concur with Jon. This episode is fine to proceed, but let's try to make sure future episodes have a firm connection to Rust.
